content/en/docs/reference/config/analysis/ist0109/index.md

```diff
@@ -3,10 +3,90 @@ title: ConflictingMeshGatewayVirtualServiceHosts
 layout: analysis-message
 ---
 
-This message occurs when Istio detects an overlap between virtual service
-resources. For example, you have multiple virtual services defined to use the
-same hostname and gateway, where the gateway is set to `mesh` or no gateway
-specified.
+This message occurs when Istio detects an overlap between
+[virtual service](/docs/reference/config/networking/virtual-service)
+resources that conflict with one another. For example, multiple virtual
+services defined to use the same hostname and attached to mesh gateway
+will generate this message. Note that Istio supports merging of virtual
+services that are attached to the ingress gateways.
 
-To resolve this problem, check your Istio configuration, remove the conflicting
-values and try again.
+## Resolution
+
+To resolve this issue, you can take one of the following actions:
+
+* Merge the conflicting virtual services into a single resource.
+* Make the hostnames unique across virtual services attached to mesh gateway.
+* Scope the resource to a specific namespace by setting the `exportTo` field.
+
+## Examples
+
+The following `VirtualServices`, `productpage` in namespaces `team1` and `team2`
```

Comment:
I found this sentence confusing to read at first. Here's a stab at an alternative:
```
The `productpage` virtual service in namespace `team1` conflicts with the `productpage` 
virtual service in namespace `team2` because both of the following are true:
  * they are attached to the default mesh gateway (no custom gateway is specified)
  * They have the same host `productpage.default.svc.cluster.local` defined.
```

It may also be wise to have a different name for each virtual service, so it's clearer that the conflict isn't that they share a name (which is fine) but that they share a host.